ci/dib_image/build-centos-dev-image.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -eux
+
+export IMAGE_OS="centos"
+
+# Install disk-image-builder
+sudo dnf install python3 pip -y
+sudo pip install diskimage-builder
+
+source dib-and-image-vars.sh
```

Comment:
```suggestion
. dib-and-image-vars.sh
```
`source` is deprecated, use `.` for the same.